imports/ui/explorer-components/unit-group-list.jsx

```diff
@@ -54,7 +54,7 @@ export class UnitGroupList extends Component {
             <div className='flex items-center h3 bt b--light-gray bg-white'
               onClick={evt => this.handleExpandUnit(evt, unitTitle)}
             >
-              <UnitTypeIcon iconInExplorer={unitType} />
+              <UnitTypeIcon iconInExplorer={unitType} cases />
```

Comment:
It would be a better solution to add the margin to the component directly or wrap in a div that has the margin, rather than using an obscure boolean prop for doing it